packages/cardhost/app/templates/components/cards/default/isolated.hbs

```diff
@@ -24,4 +27,10 @@
     {{/if}}
     <br>
   {{/each}}
+  {{#if @dropField}}
+    <DropZone
+      @dropField={{fn @dropField @card.isolatedFields.length}}
+      @position={{@card.isolatedFields.length}}
```

Comment:
is there a reason why we need to pass the position in twice for the `<DropZone>` Component? why cant we leverage the `this.args.position` in the `dropField()` action? is there  a reason that those positions would ever be different?